README.md

```diff
@@ -57,12 +69,15 @@ jobs:
 
 See Slack's [Basic message formatting](https://api.slack.com/docs/message-formatting) documentation for guidance on formatting notification messages.
 
-- ### Status
+### Status
+Send a status alert at the end of a job based on success or failure. This must be the last step in a job.
 
-|  Usage | slack/status   |
-| ------------ | ------------ |
-| **Description:** | Send a status alert at the end of a job based on success or failure. This must be the last step in a job. |
-|  **Parameters:** | -  **webhook:** Enter either your Webhook value or use the CircleCI UI to add your token under the `SLACK_WEBHOOK` environment variable <br> <br> - **fail_only:** `false` by default. If set to `true, successful jobs will _not_ send alerts <br> <br> - **mentions:**  comma separated list of Slack user IDs, or Group (SubTeam) IDs. example 'USER1,USER2,USER3'. Note, these are Slack User IDs, not usernames. The user ID can be found on the user's profile. Look below for infomration on obtaining Group ID. <br> <br> - **only_for_branch**: Optional: If set, a specific branch for which status updates will be sent. |
+| Parameter | Type | Default | Description |
+|-----------|------|---------|-------------|
+| `webhook` | `string` | ${SLACK_WEBHOOK} | Enter either your webhook value or use the CircleCI UI to add your token under the `SLACK_WEBHOOK` environment variable |
+| `mentions` | `string` |  | Comma-separated list of Slack User or Group (SubTeam) IDs (e.g., "USER1,USER2,USER3"). _**Note:** these are Slack User IDs, not usernames. The user ID can be found on the user's profile. Look below for information on obtaining Group ID._ |
+| `fail_only` | `string` | false | If set to "true," successful jobs will _not_ send alerts |
+| `only_for_branch` |  | ${SLACK_WEBHOOK} | If set, a specific branch for which slack status updates will be sent |
```

Comment:
Since there is no default for `only_for_branch:
https://github.com/CircleCI-Public/slack-orb/blob/3057745546bd905ff4eaca2c7682452ebab823df/src/commands/status.yml#L21-L24

should this be:
```suggestion
| `only_for_branch` |  |  | If set, a specific branch for which slack status updates will be sent |
```
?

Reply:
ugh, oops, thank you !